<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EM1500_settled">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Settled</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>E-Motion 1500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EM1500_neutral">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Neutral</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EM1500_engaged">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Engaged</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EM8000_engaged">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Engaged</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EM8000_neutral">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Neutral</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EM8000_settled">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Settled</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>E-Motion 8000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="COMAU_engaged">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Engaged</string>
    </property>
   </widget>
   <widget class="QPushButton" name="COMAU_settled">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Settled</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COMAU</string>
    </property>
   </widget>
   <widget class="QSlider" name="COMAU_omega">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>41</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="COMAU_engaged_fast">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To Engaged - Fast</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Omega</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="pyqtgraph_plot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>571</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
